isthmus/src/main/java/io/substrait/isthmus/expression/AggregateFunctionConverter.java

```diff
@@ -76,7 +78,13 @@ public Optional<AggregateFunctionInvocation> convert(
       AggregateCall call,
       Function<RexNode, Expression> topLevelConverter) {
 
-    FunctionFinder m = signatures.get(call.getAggregation());
+    // replace COUNT() + distinct == true and approximate == true with APPROX_COUNT_DISTINCT
+    // before converting into substrait function
+    SqlAggFunction aggFunction = call.getAggregation();
+    if (aggFunction == SqlStdOperatorTable.COUNT && call.isDistinct() && call.isApproximate()) {
+      aggFunction = SqlStdOperatorTable.APPROX_COUNT_DISTINCT;
```

Comment:
Does this mean we can go from sql -> plan for approx_count_distinct?

Reply:
Yes, now we can go sql -> substrait for approx_count_distinct.  Previously, in sql -> substrait, the code will convert `approx_count_distinct` to `count`, which is not correct.